Trustroots/trustroots
# Add Tooltip component

#### Proposed Changes

* create `Tooltip`, a reusable component that adds tooltip to the wrapped component

```jsx
<Tooltip tooltip="text"><WrappedComponent></Tooltip>
```

##### Usage

* Read `modules/core/client/components/README.md`

#### Testing Instructions

* `git switch with-tooltip`
* test tooltip in tribes `JoinButton`:
  - log in
  - go to `/tribes`
  - see that the tooltip under Joined button works
    ![image](https://user-images.githubusercontent.com/7449720/78501854-0a338080-775e-11ea-8652-f56cdc0cb79b.png)

* test tooltip in offers
  - log in
  - go to my profile
  - see that the tooltip next to Accommodation edit button works
    ![image](https://user-images.githubusercontent.com/7449720/78501816-cb052f80-775d-11ea-92ef-0b0a4747253f.png)

Preparation for #1350
